src/System.Private.CoreLib/src/System/Decimal.cs

```diff
@@ -312,7 +312,7 @@ int IComparable.CompareTo(Object value)
             if (value == null)
                 return 1;
             if (!(value is Decimal))
-                throw new ArgumentException(SR.Arg_MustBeDecimal);
+                throw new ArgumentException(SR.Arg_MustBeDecimal, nameof(value));
```

Comment:
I would prefer to be consistent with CoreCLR. If you believe that this is a good change to make - make it in CoreCLR first and then port it to CoreRT. It is better to do BCL improvements in CoreCLR first because of we have better test coverage there.